proxy/api/src/main.rs

```diff
@@ -70,67 +157,20 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         });
         let config = coco::config::configure(paths, key, *coco::config::LOCALHOST_ANY, seeds);
 
-        coco::into_peer_state(config, signer.clone()).await?
+        coco::into_peer_state(config, signer.clone(), store.clone(), RunConfig::default()).await?
     };
 
-    if args.test {
-        let state = state.lock().await;
-        // TODO(xla): Given that we have proper ownership and user handling in coco, we should
```

Comment:
It did not make any difference whether or not those were set up, so I removed it.